editor/file_system/dependency_editor.h

```diff
@@ -39,16 +39,38 @@ class PopupMenu;
 class Tree;
 class TreeItem;
 class VBoxContainer;
+class LineEdit;
+class MenuButton;
+
+enum class DependencyEditorSortBy {
+	TYPE,
+	TYPE_REVERSE,
+	NAME,
+	NAME_REVERSE,
+	PATH,
+	PATH_REVERSE,
+	MAX,
+};
+
+enum class DependencyEditorColumn {
```

Comment:
The names don't need to be that verbose.
```suggestion
enum class Column {
```
Like, in outside code typically you'd write `DependencyEditor::Column`, repeating the class in the enum name is redundant.